src/annotations/AnnotationManager.php

```diff
@@ -299,6 +306,21 @@ protected function getAnnotations($class_name, $member_type = self::MEMBER_CLASS
                 }
             }
 
+            if ($this->_traitsSupported && !isset($file->data[$key]) && $member_name !== null) {
+                $traitAnnotations = array();
+
+                if (isset($file) && isset($file->traitMethodOverrides[$class_name][$member_name])) {
```

Comment:
> `isset($file->traitMethodOverrides[$class_name][$member_name])`

Doesn't this trigger a notice, when `$file->traitMethodOverrides[$class_name]` isn't an array?